scripts/_helpers_dist.py

```diff
@@ -48,6 +51,43 @@ def sets_path_to_root(root_directory_name):
             upper_path = os.path.dirname(os.path.abspath("."))  # name of upper folder
             os.chdir(upper_path)
```

Comment:
An interesting trick to avoid duplicating code from pypsa-earth to here could be something like:

``` python
import sys
sys.path.append("../pypsa-earth/scripts")

import helpers as pe_helpers
```
This fragment above to be placed at the beginning of the file and then the following below:

``` python
handle_exception = pe_helpers.handle_exception
create_logger = pe_helpers.create_logger
read_osm_config = pe_helpers.read_osm_config
```

That should not need to download the osm_config file too.
Could you try?

Reply:
okay I did a run and it seems to work! I just have to change the path instead of "../pypsa-earth/scripts" i use "pypsa-distribution\pypsa-earth\scripts", beacause with "../pypsa-earth/scripts" it gave me error